<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LoadWindow</class>
 <widget class="QDialog" name="LoadWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>582</width>
    <height>132</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="pathLbl">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>111</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Select load file:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="pathEdit">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>50</y>
     <width>351</width>
     <height>28</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="loadGameBtn">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>90</y>
     <width>84</width>
     <height>28</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(138, 226, 52);</string>
   </property>
   <property name="text">
    <string>Load game</string>
   </property>
  </widget>
  <widget class="QPushButton" name="browseBtn">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>50</y>
     <width>84</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Browse</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
